files/system/usr/share/ublue-os/firstboot/yafti.yml

```diff
@@ -19,6 +19,15 @@ screens:
       actions:
         - run: ujust harden-flatpak
 
+  can-we-switch-to-signed:
```

Comment:
neither is this https://github.com/secureblue/secureblue/pull/564/files#diff-d48867bc4fd1881c03a2b2fee68a985512e7633905af8ea05fd5f62ccb8021ee